SimpleInventoryManagementSystem/Program.cs

```diff
@@ -1,13 +1,63 @@
 ﻿
+using SimpleInventoryManagementSystem;
 using System;
 
-namespace SimpleInventoryManagementSystem
+namespace SimpleInventoryManagementSystem;
+
+public class Program
 {
-    public class Program
+    enum Services{
```

Comment:
A nitpick, but let's stick to C# conventions as most as we can.

```suggestion
    enum Services
    {
```